hypha/apply/funds/tables.py

```diff
@@ -190,16 +190,16 @@ def get_used_funds(request):
     return Page.objects.filter(applicationsubmission__isnull=False).distinct()
 
 
-def get_round_leads(request):
+def get_round_leads():
     return User.objects.filter(submission_lead__isnull=False).distinct()
 
 
-def get_reviewers(request):
+def get_reviewers():
     """ All assigned reviewers, staff or admin """
     return User.objects.filter(Q(submissions_reviewer__isnull=False) | Q(groups__name=STAFF_GROUP_NAME) | Q(is_superuser=True)).distinct()
 
 
-def get_screening_statuses(request):
```

Comment:
@vimal1083, do not edit these methods. Rather create new methods in the api directory, and use those for the api at both places(For filter class and filter options api).

The idea is to not be dependent django view methods for api views.

Also editing old django view methods would break existing changes.